tests/monster_test.fbs

```diff
@@ -84,6 +87,19 @@ table Monster {
   parent_namespace_test:InParentNamespace (id:34);
 }
 
+table TestAttributeTable (invincible) {
```

Comment:
this is going to generate a lot of code in all languages, for a small feature that currently only affects C++. Please instead add a bunch of attributes to an existing field.